src/main/java/com/programmers/vouchermanagement/controller/rest/RestCustomerController.java

```diff
@@ -0,0 +1,45 @@
+package com.programmers.vouchermanagement.controller.rest;
+
+import com.programmers.vouchermanagement.dto.CustomerDto;
+import com.programmers.vouchermanagement.service.CustomerService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/api/v1/customer")
```

Comment:
REST 규칙을 따른다면 복수 형태가 되어야하지 않을까요?
```suggestion
@RequestMapping("/api/v1/customers")
```